### PR TITLE
Add Koala analytics integration docs

### PR DESCRIPTION
**Pull request by hanywang2**
## Summary

Update the analytics docs with Koala's documentation and re-organized the cards

### CODE REVIEW

File: docs/analytics/set-up/koala.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Koala"
+---
+
+Add the following to your `mint.json` file to send analytics to Mixpanel.
```
Comment (hahnbeelee) [inline docs/analytics/set-up/koala.mdx:5]:
```suggestion
Add the following to your `mint.json` file to send analytics to Koala.
```

**Review by hahnbeelee — CHANGES_REQUESTED** [review]

File: docs/analytics/set-up/koala.mdx (hunk 1 above)
Comment (hanywang2) [reply docs/analytics/set-up/koala.mdx:5]: Great catch

File: docs/analytics/set-up/koala.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+<CodeGroup>
+
+```json Analytics options in mint.json
+"analytics": {
+    "koala": {
+        "projectId": "required"
+    },
```
Comment (hahnbeelee) [inline docs/analytics/set-up/koala.mdx:13]: take out the comma too, other ones dont have it and it'll be incorrect javascript if you copy paste this exactly into your mint.json

**Review by hahnbeelee — APPROVED** [review]
LGTM

**Pull request merged**